userapi/src/main/java/fk/prof/userapi/api/AggregatedProfileLoader.java

```diff
@@ -28,6 +32,12 @@
  */
 public class AggregatedProfileLoader {
 
+    private static String profileLoadTimerPrefix = "profile.load.timer";
```

Comment:
If I understand correctly, this class has nothing to do with async execution. Yet it is littered with Futures. Let's use this PR as an opportunity to make this code readable.

Reply:
calls are meant to be done in async way. Refer to comment above.

Reply:
Sure, let caller worry how to call these methods. If it calls them in an async fashion, it can wrap the result of these methods in a future. Meanwhile, as discussed offline, lets make this class simpler